src/EditorFeatures/Core.Wpf/InlineRename/InlineRenameSession.cs

```diff
@@ -92,7 +92,7 @@ private set
         /// This task is a continuation of the allRenameLocationsTask that is the result of computing
```

Comment:
```suggestion
        /// This task is a continuation of the <see cref="_allRenameLocationsTask" /> that is the result of computing
```

Reply:
(not your fault, but should hopefully simplify the task of understanding this code in the future...)

Reply:
Fixed